qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml

```diff
@@ -0,0 +1,22 @@
+tasks:
+- install:
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      force-repo: https://github.com/soumyakoduri/s3-tests.git
```

Comment:
This will be removed once https://github.com/ceph/s3-tests/pull/421 gets merged.